srcalc/src/main/java/gov/va/med/srcalc/util/RetrievedValue.java

```diff
@@ -0,0 +1,51 @@
+package gov.va.med.srcalc.util;
+
+import java.util.Date;
+
+/**
+ * A class that will store the value of a retrieved field as well as the
+ * date on which that field was measured. The value and date should not change
+ * after they are stored. The class currently only supports the double type.
```

Comment:
I like that you made this immutable. Instead of saying "should not change", why not just be strong and say these objects are immutable?